### ISSUE TITLE
make adamantoise not spin in first person flight?

### ISSUE THREAD

**Issue by wallminter** [WesleyLuk90/ffxiv-vr#27]
this is really niche and low-priority, but the adamantoise mount spins around when you fly with it, and it's really weird in first person. luckily the camera doesn't spin around with it. it might be nice if possible to make it so the adamantoise just stays facing forward when you fly in first person.

EDIT: thought about it more and this might just be something that would belong better in the Simple Tweaks dalamud plugin? maybe not VR-specific enough to include in this plugin

**Comment by qq1980-cn** [comment]
isn't spinning the purpose of the admantoise mount?

**Comment by fdsimms** [comment]
> isn't spinning the purpose of the admantoise mount?

Yeah it just looks and feels bad in first person VR, like I said. It's not a big deal but it makes it so I don't want to use this mount in VR even though it looks great on the ground.
